base/server/scripts/operations

```diff
@@ -1396,228 +1326,6 @@ EOF
-            cd \"$CATALINA_BASE\"; \
-            \"$CATALINA_SH\" $@"
-
-    if [ "$AUTHBIND" = "yes" -a "$1" = "start" ]; then
-            TOMCAT_SH="'$TOMCAT_SH'"
-    fi
-
-    # Run the catalina.sh script as a daemon
-    if [ ! -f "$CATALINA_BASE"/logs/catalina.out ]; then
-        # run install as pkiuser to work around #841371
-        su $TOMCAT_USER -s /bin/bash -c "install -m 644 /dev/null $CATALINA_BASE/logs/catalina.out"
-    fi
-    install -o $TOMCAT_USER -m 644 /dev/null "$CATALINA_PID"
-    start-stop-daemon --start -b -u "$TOMCAT_USER" -g "$TOMCAT_GROUP" \
-            -c "$TOMCAT_USER" -d "$CATALINA_TMPDIR" -p "$CATALINA_PID" \
-            -x /bin/bash -- -c "$AUTHBIND_COMMAND $TOMCAT_SH"
-    status="$?"
-    set +a
-    return $status
-}
-
-set_debian_tomcat_parameters()
-{
-    set_java_home
-    CATALINA_HOME=/usr/share/tomcat8
-    CATALINA_BASE=/var/lib/pki/${PKI_INSTANCE_NAME}
-    DESC="Tomcat instance for ${PKI_INSTANCE_NAME}"
-
-    if [ -z "$JAVA_OPTS" ]; then
-        JAVA_OPTS="-Djava.awt.headless=true"
-    fi
-
-    # overwrite with settings from the registry file
-    if [ -f /etc/default/${PKI_INSTANCE_NAME} ]; then
-        . /etc/default/${PKI_INSTANCE_NAME}
-    fi
-
-    JVM_TMP=/tmp/tomcat-${PKI_INSTANCE_NAME}-tmp
-    if [ -z "$CATALINA_TMPDIR" ]; then
-        CATALINA_TMPDIR="$JVM_TMP"
-    fi
-
-    # Set the JSP compiler if set in the tomcat8.default file
-    if [ -n "$JSP_COMPILER" ]; then
-        JAVA_OPTS="$JAVA_OPTS -Dbuild.compiler=\"$JSP_COMPILER\""
-    fi
-
-    # Generate catalina.policy dynamically.
-    # for now, do not generate policy here, because we need to figure out
-    # where to get the tomcat policy.
-
-    TOMCAT8_SECURITY=no
-    SECURITY=""
-    if [ "$TOMCAT8_SECURITY" = "True" ]; then
-        SECURITY="-security"
-    fi
-
-    # Define other required variables
-    CATALINA_PID="/var/run/${PKI_INSTANCE_NAME}.pid"
-    CATALINA_SH="$CATALINA_HOME/bin/catalina.sh"
-
-    # Look for Java Secure Sockets Extension (JSSE) JARs
-    if [ -z "${JSSE_HOME}" -a -r "${JAVA_HOME}/jre/lib/jsse.jar" ]; then
-        JSSE_HOME="${JAVA_HOME}/jre/"
-    fi
-}
-
-# used to stop debian instances
-stop_instance()
-{
-    set_debian_tomcat_parameters
-    log_daemon_msg "Stopping $DESC" "${PKI_INSTANCE_NAME}"
-
-    if [ -f "$CATALINA_PID" ]; then
-        start-stop-daemon --stop --pidfile "$CATALINA_PID" \
-                --user "$TOMCAT_USER" \
-                --retry=TERM/20/KILL/5 >/dev/null
-        if [ $? -eq 1 ]; then
-            log_progress_msg "$DESC is not running but pid file exists, cleaning up"
-        elif [ $? -eq 3 ]; then
-            PID="`cat $CATALINA_PID`"
-            log_failure_msg "Failed to stop $PKI_INSTANCE_NAME (pid $PID)"
-            exit 1
-        fi
-        rm -f "$CATALINA_PID"
-        rm -rf "$JVM_TMP"
-    else
-        log_progress_msg "(not running)"
-    fi
-    log_end_msg 0
-}
-
-stop()
```

Comment:
Just to confirm, is this a Debian-specific code?

Reply:
yup